testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java

```diff
@@ -101,13 +101,14 @@ void handleInteractiveRequestRedirect(int requestCode, int resultCode, Intent da
     // The app does nothing with the newly acquired scopes yet.
     void doExtraScopeRequest(final Activity activity) {
         getUsers();
+        mApplication.acquireToken()
         mApplication.acquireToken(activity,
                 SCOPES,
-                mUsers.get(0), // The user object
+                "",
                 UiBehavior.CONSENT,
                 "", // Extra query parameters
                 EXTRA_SCOPES,
-                "", // AuthorityMetadata
+                null,
                 new AuthenticationCallback() {
                     @Override
                     public void onSuccess(AuthenticationResult authenticationResult) {
```

Comment:
Can't see it in the diff, but line `134` contains a call to `getUsers()` -- needs to be removed or adapted to use `getAccount()`

Reply:
Corrected in the PR you sent.